x-pack/plugins/maps/public/classes/sources/es_search_source/scaling_form.tsx

```diff
@@ -194,16 +272,31 @@ export class ScalingForm extends Component<Props, State> {
       );
     }
 
-    let scalingForm = null;
+    let topHitsOptionsForm = null;
     if (this.props.scalingType === SCALING_TYPES.TOP_HITS) {
-      scalingForm = (
+      topHitsOptionsForm = (
         <Fragment>
           <EuiHorizontalRule margin="xs" />
           {this._renderTopHitsForm()}
         </Fragment>
       );
     }
 
+    let mvtDisclaimer = null;
```

Comment:
Is this disclaimer really needed? I think something like this is better for the docs to explain when/why to use something but just adds clutter to the UI. I would recommend removing or putting into a tooltip for the option itself.